lms/templates/courseware/courseware.html

```diff
@@ -89,31 +92,10 @@
   % endif
 
   <script type="text/javascript">
-    var $$course_id = "${course.id | escapejs}";
-
-    $(function(){
-        $(".ui-accordion-header a, .ui-accordion-content .subtitle-name").each(function() {
```

Comment:
Ah, I didn't actually try it out. I just ran the same experiment as you and couldn't match any of these classes. It looks like ``ui-accordion-header`` is a JQuery UI component, so maybe we switched it out at some point. I now think you should remove both this code and the equivalent logic in courseware-chromeless.html (which never even had an accordion!).